AK/Tuple.h

```diff
@@ -135,6 +129,21 @@ struct Tuple : Detail::Tuple<Ts...> {
     }
 
     Tuple& operator=(Tuple const& other)
+    {
+        if (&other == this)
+            return *this;
+        set(other, Indices());
+        return *this;
+    }
+
+    template<typename... Us>
+    Tuple& operator=(Tuple<Us...>&& other)
+    {
```

Comment:
I'd rather not add logic to allow using references in Tuple without a real use case in user/library code that needs it